cluster/gce/gci/configure-helper.sh

```diff
@@ -3098,10 +3098,10 @@ oom_score = -999
   runtime_type = "io.containerd.runc.v2"
 [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
   endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]
-# Enable registry.k8s.io as the primary mirror for k8s.gcr.io
+# Enable registry.k8s.io as the primary mirror for registry.k8s.io
```

Comment:
don't replace it here please. This is a mirror definition that will redirect from `k8s.gcr.io` to `registry.k8s.io` as intended